### ISSUE TITLE
Bluetooth Support

### ISSUE THREAD

**Issue by gibbi** [georgerobotics/cyw43-driver#7]
i might be mistaken but so far there seems to be no support for the Bluetooth in the Pico W as of now. Is this planned?

**Comment by dpgeorge** [comment]
That is correct, there is currently no support for Bluetooth in this driver, or on the Pico W.

**Comment by peterharperuk** [comment]
Out of interest - what would you use it for?

**Comment by gibbi** [comment]
i would like to have a Pico W a standalone receiver for MiThermometers(https://github.com/atc1441/ATC_MiThermometer), as well as potentially other IoT Sensors. Another application would be mechanical keyboards, Remotes for TVs.. there is a lot that could be done.

**Comment by tralamazza** [comment]
Is it explicitly disabled in the blob?

**Comment by peterharperuk** [comment]
No. But it's not a simple matter of enabling / disabling a flag

**Comment by Ferdi265** [comment]
bluetooth support seems to be being worked on:

initial firmware and low-level driver code was merged recently in #60

**Comment by XZCE** [comment]
> We would have to come up with a test to reproduce the problem if we want to raise it with infineon.

This is my best guess for the code, using your example picow_tcp_server, with iterations changed from 10 to 10000 - I assume the client example is this tests companion - so would need it changed there also.  I'm guessing this is a 2 PicoW test.  I have not used the Pico SDK for a year now, so this is untested (not even test-compiled) so the BT #include paths may be wrong for that environment.  They were correct for the Arduino environment.  I have no idea how to enable the BT library stuff in the TCP example, some sort of magical incantation in the cmake texts I expect.

I've put the 30 second wait to start BT in the run loop, where you have a convenient 1 second wait between iterations, then at 33 seconds it starts the BT device connect.

[picow_tcp_server.c](https://github.com/user-attachments/files/23972788/picow_tcp_server.c)
